apps/condo/bin/prepare.js

```diff
@@ -0,0 +1,35 @@
+const { getAppEnvValue, getAppServerUrl, updateAppEnvFile, prepareAppEnvLocalAdminUsers } = require('@open-condo/cli')
+const { getRandomString } = require('@open-condo/keystone/test.utils')
+
+async function updateAppEnvAddressSuggestionConfig (serviceName) {
+    const addressServiceUrl = await getAppServerUrl('address-service')
+    await updateAppEnvFile(serviceName, 'ADDRESS_SERVICE_URL', addressServiceUrl)
+
+    const config = {}
+    const currentValue = await getAppEnvValue(serviceName, 'ADDRESS_SUGGESTIONS_CONFIG')
+    if (currentValue) {
+        const currentValueJson = JSON.parse(currentValue)
+        if (currentValueJson.apiUrl) config.apiUrl = currentValueJson.apiUrl
+        if (currentValueJson.apiToken) config.apiToken = currentValueJson.apiToken
+    }
+    if (!config.apiUrl) config.apiUrl = `${addressServiceUrl}/suggest`
```

Comment:
The `ADDRESS_SUGGESTIONS_CONFIG` variable is a deprecated config for dadata API. It must refer to real dadata API URL because it has a different query structure.
Dadata API parses `body.query` of post-request, but the address service looks for get/post parameter called `s`.
So, it looks like the serverSideAddressApi won't work.
<img width="1011" alt="Screenshot 2023-03-04 at 11 07 00 AM" src="https://user-images.githubusercontent.com/25384290/222879146-25528d1e-f363-4c55-986b-d5a97c80bf60.png">

This variable must have the next value
```
ADDRESS_SUGGESTIONS_CONFIG={"apiUrl": "https://suggestions.dadata.ru/suggestions/api/4_1/rs/suggest/address", "apiToken": "<some-token>"}
```